src/ensembl/src/content/app/species/components/species-stats/SpeciesStats.scss

```diff
@@ -10,15 +10,22 @@ $secondary-text-color: $dark-grey;
 }
 
 .preLabel {
-  color: $secondary-text-color;
+  color: $black;
+  padding-right: 4px;
+  font-weight: $light;
 }
 
 .preLabel + .label {
   margin-top: -3px;
 }
 
+.questionButton {
+  top: 1px;
```

Comment:
Sorry, couldn't test it; but is it correct that the `top` value here is 1px, but in SpeciesMainView.scss it's 2px?

Reply:
We can't test this we dont have a scenario yet where we have a questionButton on expanded view. But I have made the value same in both places for the time being.